packages/pulse-core/package.json

```diff
@@ -17,5 +17,8 @@
     "type": "git",
     "url": "https://github.com/pulse-framework/pulse"
   },
-  "gitHead": "df2d02d8e5a07ded7bb792c0d8e1473c5b6159ba"
+  "gitHead": "df2d02d8e5a07ded7bb792c0d8e1473c5b6159ba",
```

Comment:
This isn't your fault, but a note for us — gitHead [is not supposed to be versioned](https://github.com/lerna/lerna/issues/1880#issuecomment-456160357) into the repo. I'll remove all instances of it before merging and find out why it got committed.